Software/app/src/main/java/org/foi/emp/collegecoursemanager/Adapters/KolegijAdapter.java

```diff
@@ -0,0 +1,112 @@
+package org.foi.emp.collegecoursemanager.Adapters;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.RequiresApi;
+import androidx.recyclerview.widget.RecyclerView;
+
+import org.foi.emp.collegecoursemanager.DodavanjeBodovaKolegiju;
+import org.foi.emp.collegecoursemanager.R;
+import org.foi.emp.core.Database.Database;
+import org.foi.emp.core.Entities.ElementModelaPracenja;
+import org.foi.emp.core.Entities.ElementiNaModeluPracenja;
+import org.foi.emp.core.Entities.Kolegij;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class KolegijAdapter extends RecyclerView.Adapter<KolegijAdapter.KolegijHolder> {
+    private List<Kolegij> sviKolegiji=new ArrayList<>();
+    private Context context;
+
+    public KolegijAdapter(Context context) {
+        this.context = context;
+    }
+
+    public void setKolegiji(List<Kolegij> sviProslijedeniKolegiji) {
+        sviKolegiji = sviProslijedeniKolegiji;
+    }
+
+    @NonNull
+    @Override
+    public KolegijAdapter.KolegijHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
+        View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.item_kolegij, parent, false);
+        return new KolegijAdapter.KolegijHolder(view);
+    }
+
+    @RequiresApi(api = Build.VERSION_CODES.N)
+    @Override
+    public void onBindViewHolder(@NonNull KolegijAdapter.KolegijHolder holder, final int position) {
+        final Kolegij trenutniKolegij = sviKolegiji.get(position);
+        String brojBodovaOd100="";
```

Comment:
Check if you have StringUtils class and then make it as `String brojBodovaOd100 = StringUtils.EMPTY;`